File: Modul5/PRAK502-2310817110008-Muhammad_Raihan.c
#include <stdio.h>

int mutlak(int angka){
    if (angka < 0) {
        return angka*-1;
    }
    else {
        return angka;
    }
}
int hitung(int nilai1, int nilai2){
    return mutlak(nilai1-nilai2);
}
int main()
{
int a,b,c,d;
    scanf("%d %d %d %d",&a,&c,&b,&d);
    int Hasil = hitung(a,b) + hitung(c,d);
    printf("%d", Hasil);
    return 0;
}

File: Modul5/PRAK502-2310817110008-Muhammad_Raihan.py
def mutlak(x):
    if x<0: return x*-1
    else: return x
def hitung(y,z):
    return mutlak(y-z)

a,c,b,d = map(int, input().split())
hasil = hitung(a,b)+hitung(c,d)
print(hasil)